MUXSDKImaListener/Classes/MuxImaListener.m

```diff
@@ -95,6 +98,16 @@ - (MUXSDKAdEvent *_Nullable) dispatchEvent:(IMAAdEvent *)event {
             [_playerBinding dispatchAdEvent: playbackEvent];
             playbackEvent = [MUXSDKAdPlayingEvent new];
             break;
+        case kIMAAdEvent_LOG:
+            if (adData && adData[@"logData"]) {
+                NSDictionary *errorLog = (NSDictionary *)adData[@"logData"];
+                if (errorLog) {
+                    if (errorLog[@"errorCode"] || errorLog[@"errorMessage"] || errorLog[@"type"]) {
```

Comment:
Are these keys coming from IMA? (If so wondering if we can add a doc link if there is one to help future us)

Reply:
They come from IMA, yes but there's no public constants for them. I got these values by failing requests with proxyman then dumping events the IMASDKEvents to the logs

Reply:
<img width="509" alt="Screenshot 2024-04-04 at 3 29 17 PM" src="https://github.com/muxinc/mux-stats-google-ima/assets/319221/c9144042-6872-4dd5-977b-eeb036130df0">

There's a log snippet of the keys above ⬆️

Reply:
Got it!